packages/cli/src/Interfaces.ts

```diff
@@ -124,13 +123,13 @@ export interface IExecutionBase {
 	retryOf?: string; // If it is a retry, the id of the execution it is a retry of.
 	retrySuccessId?: string; // If it failed and a retry did succeed. The id of the successful retry.
 	status: ExecutionStatus;
+	waitTill?: Date | null;
```

Comment:
Moving waitTill to this type since it's an attribute that could always exist.

Reply:
Let's remove it then from the interfaces that extend from this? e.g. `IExecutionFlattedDb` and `waitTill`

Reply:
Good finding, removed.

Reply:
1. There are a few more redundancies like `retryOf` and `retrySuccessId` in `IExecutionResponse`, but we might want to clean these up later?
2. `IExecutionBase` has `workflowId?: string`, is this correct?

Reply:
Yup I'll create separate tickets for these as it's not in the extent of this PR and don't really hurt